### PR TITLE
Replaced the 'Defer' function with the new Promise

### PR DESCRIPTION
**Pull request by sipmann**
It will break the backward compatibility but will be more accurate

### CODE REVIEW

File: locache.js [hunk 1]
```diff
@@ -104,6 +108,7 @@
     // Internal utility functions
     // --------------------
 
+    // @Deprecated
```
Comment (d0ugal) [inline locache.js]: I'd rather see this just deleted if it isn't used internally anymore. It isn't part of the public API, so we don't need to support it.

**Comment by sipmann** [comment]
Yeap sir. done.

**Comment by d0ugal** [comment]
Great, I gave you commit access so you should be able to merge it when you are ready.

**Comment by sipmann** [comment]
yup, for some reason i still can't merge this pull. can u check it again?

**Comment by d0ugal** [comment]
You need to accept the invitation to the repo. I'll merge it for now.

**Pull request merged**